tools/scp/scp_workflow.xml

```diff
@@ -0,0 +1,503 @@
+<tool id="scp_full_workflow" name="scp: Single Cell Proteomics data analysis" version="@TOOL_VERSION@+galaxy0" profile="23.0">
```

Comment:
```suggestion
<tool id="scp_full_workflow" name="scp" version="@TOOL_VERSION@+galaxy0" profile="23.0">
```

Reply:
please use only scp as the name or remove the description. Otherwise, it will be a very long name in the tool panel.

Reply:
I think that *bioconductor-scp* is a good name. Only scp is indeed very/too vague.